cg/meta/transfer/flowcell.py

```diff
@@ -1,212 +0,0 @@
-"""API for transfer a flow cell."""
-
-import logging
-from datetime import datetime
-from pathlib import Path
-from typing import List, Optional
-
-from housekeeper.store.models import Bundle, Version
-from cg.apps.cgstats.stats import StatsAPI
-from cg.apps.housekeeper.hk import HousekeeperAPI
-from cg.constants import FlowCellStatus
-from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
-from cg.constants.housekeeper_tags import SequencingFileTag
-from cg.models.cgstats.flowcell import StatsFlowcell
-from cg.store import Store
-from cg.store.models import Sample, Flowcell
-
-LOG = logging.getLogger(__name__)
-
-
-def _set_status_db_sample_sequenced_at(
-    status_db_sample: Sample, flow_cell_sequenced_at: datetime
-) -> None:
-    """Set sequenced at for sample in status db."""
-    is_newer_date: datetime = (status_db_sample.sequenced_at is None) or (
-        flow_cell_sequenced_at > status_db_sample.sequenced_at
-    )
-    if is_newer_date:
-        status_db_sample.sequenced_at: datetime = flow_cell_sequenced_at
-
-
-def log_enough_reads(
-    status_db_sample_reads: int, application_expected_reads: int, cgstats_sample_name: str
-) -> None:
-    """Check and log if sample in status db got enough reads."""
-    enough_reads: bool = status_db_sample_reads > application_expected_reads
-    LOG.info(f"Added reads to sample: {cgstats_sample_name} - {status_db_sample_reads} ")
-    LOG.info(f"[{'DONE' if enough_reads else 'NOT DONE'}]")
-
-
-class TransferFlowCell:
```

Comment:
this functionality is in the new dempux post processing workflow. This API is technically not needed. That being said it would remove to option to transfer a flow cell using the `cg transfer` cmd.